### PR TITLE
ordered_associative_task SChyruk

### PR DESCRIPTION
**Pull request by ChNokia**

### CODE REVIEW

File: ordered_associative_task/student/include/impl.h [hunk 1]
```diff
@@ -15,4 +15,21 @@
...
+        {49, 2},
+        {69, 3},
+        {89, 4},
+        {90, maxMark}
+    };
+
+    auto reultIter = pointSystem.lower_bound(markIn100Point);
+
+    if (reultIter == pointSystem.cend()) {
+        return maxMark;
```
Comment (smay1613) [inline ordered_associative_task/student/include/impl.h:31]: please solve this task only with map, without special values

File: ordered_associative_task/student/include/impl.h [hunk 2]
```diff
@@ -15,4 +15,21 @@
  *
  * @return 1-5
  */
-size_t convertMarkTo5PointSystem(size_t markIn100Point);
+size_t convertMarkTo5PointSystem(size_t markIn100Point) {
+    int maxMark = 5;
+    std::map<int, int> pointSystem {
```
Comment (smay1613) [inline ordered_associative_task/student/include/impl.h:20]: make static const

File: ordered_associative_task/student/include/impl.h (hunk 1 above)
Comment (ChNokia) [reply ordered_associative_task/student/include/impl.h]: done

File: ordered_associative_task/student/include/impl.h [hunk 3]
```diff
@@ -15,4 +15,20 @@
...
+        {29, 1},
+        {49, 2},
+        {69, 3},
+        {89, 4},
+        {90, 5}
+    };
+
+    auto reultIter = pointSystem.lower_bound(markIn100Point);
+
+    if (reultIter == pointSystem.cend()) {
```
Comment (smay1613) [inline ordered_associative_task/student/include/impl.h:29]: try to avoid this check at all
hint: use upper_bound

Comment (ChNokia) [reply ordered_associative_task/student/include/impl.h]: done

**Pull request closed without merging**